server/src/test/proto/spine/test/aggregate/events.proto

```diff
@@ -28,10 +28,10 @@ option java_multiple_files = true;
 
 import "spine/annotations.proto";
 import "spine/test/aggregate/project.proto";
-import "spine/test/aggregate/task.proto";
 
 message ProjectCreated {
     ProjectId project_id = 1;
+    string name = 2;
```

Comment:
Yeah, but imagine we have the case when the client sets IDs (actually, it would be rather painful to get rid of the ID in this command, as it is the most popular one in tests).